commands/python/libs/k8s/helm.py

```diff
@@ -14,13 +14,19 @@ class Helm(object):
     PACKAGE_DIR_PATH = "helm_charts/0.0.0/{project_name}/{git_ref}/"
     PACKAGE_PATH = "helm_charts/0.0.0/{project_name}/{git_ref}/{chart_name}.{git_ref}.tgz"
 
+
+    def __init__(self, helm2):
+        self.helm = "helm2" if helm2 else "helm"
+
     @property
     def helm_home(self):
         return join(expanduser("~"), ".helm")
 
     def init(self):
-        # We need to have local helm initialized for it to works
-        subprocess.check_call(["helm", "init", "--client-only"])
+        # Only need to init on helm2. Helm2 Doesn't use tiller and therefore needs no init
+        if self.helm == "helm2":
+            # We need to have local helm2 initialized for helm2 to work
+            subprocess.check_call([self.helm, "init", "--client-only"])
```

Comment:
I'm confused how this works if we fully removed the tiller setup in `aladdin.sh`, but i'm not saying it won't.